### ISSUE TITLE
[BUG] GCS start-up error /e816T.ReInit seems to do nothing

### ISSUE THREAD

**Issue by barentine** [python-microscopy/python-microscopy#515]
**Describe the bug**
Pretty regularly hit 
```
Traceback (most recent call last):
  File "C:\Users\Bergamot\Anaconda2\envs\chaos\lib\threading.py", line 916, in _bootstrap_inner
    self.run()
  File "C:\Users\Bergamot\Anaconda2\envs\chaos\lib\threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "c:\users\bergamot\code\python-microscopy\PYME\Acquire\Hardware\Piezos\piezo_e816_dll.py", line 283, in _Loop
    self.position[0] = float(gcs.qPOS(self.id, b'A')[0])+ self.osen
  File "c:\users\bergamot\code\python-microscopy\PYME\Acquire\Hardware\GCS\gcs.py", line 102, in __call__
    self.HandleError(ID)
  File "c:\users\bergamot\code\python-microscopy\PYME\Acquire\Hardware\GCS\gcs.py", line 14, in HandleError
    raise RuntimeError('GCS error: %d - %s' % (errno, TranslateError(errno)))
RuntimeError: GCS error: -1 - b'Error during com operation (could not be specified)'
```
on start up. Piezo position is registered, but does not respond to MoveTo and e.g. moving the slider allows you to move it, but will snap back after. I often turn off the piezo (unplug effectively) then turn it back on a wait a bit, but this appears to just be superstitious as it isn't always needed - restarting PYMEAcquire can also get it back.

**To Reproduce**
1. `PYMEAcquire -i init_htsms.py`
2.  See error
3. call the piezo `ReInit` method
4. see things pause for a second
5. no error, but same broken behavior

**Platform (please complete the following information - or copy-paste from error dialog if shown):**
 - OS: [e.g. Win10] win10
 - Python version: 3.6
 - numpy version: 1.19
 - wxpython version [if a UI bug]: 4.0.4

**Additional context**
![image](https://user-images.githubusercontent.com/31105780/96007358-6877a500-0e0c-11eb-985b-b90e836a7d59.png)

**Comment by barentine** [comment]
Updates to this as I've been running up against it hard for hours each day this week:
- the error code -1 does not require a power cycle to resolve, restarting PYMEAcquire can quite often do the trick (sometimes more than once)
- I hear an audible click from the pifoc associated with this error
- the error not showing up in stdout sometimes (mentioned in #544) appears to be when another thread has stdout/err locked up. I don't see it afterwards because I ctrl-c the whole thing after the piezo stops responding

Proposed fix:
- handle error -1 either in `piezo_e816T._Loop` or more broadly in `gcs.fcnWrap.HandleError`, though that looks considerbly messier. If we catch the error -1 and beef up ReInit to flush/reconnect the COM that might do it.

**Comment by barentine** [comment]
fixed in #839
